www/class/centreonLDAP.class.php

```diff
@@ -289,7 +298,7 @@ public function replaceFilter($name)
      * @param string $username The username
      * @return string|bool The dn string or false if not found
      */
-    public function findUserDn($username)
+    public function findUserDn($username): string|bool
```

Comment:
You will not be able to cherry-pick if you use union types (only PHP 8)
```suggestion
    public function findUserDn($username)
```

Reply:
removed